pkg/cmd/server/bootstrappolicy/all.go

```diff
@@ -25,42 +25,29 @@ func Policy() *rbacrest.PolicyData {
 	}
 }
 
-func ConvertToOriginClusterRolesOrDie(in []rbac.ClusterRole) []authorizationapi.ClusterRole {
+func ConvertToOriginClusterRoles(in []rbac.ClusterRole) ([]authorizationapi.ClusterRole, error) {
 	out := []authorizationapi.ClusterRole{}
-	errs := []error{}
 
 	for i := range in {
 		newRole := &authorizationapi.ClusterRole{}
 		if err := kapi.Scheme.Convert(&in[i], newRole, nil); err != nil {
-			errs = append(errs, fmt.Errorf("error converting %q: %v", in[i].Name, err))
-			continue
+			return nil, fmt.Errorf("error converting %q: %v", in[i].Name, err)
```

Comment:
I'm not sure we should bail out on first error.seem like reporting all failures is a property that should be maintained